### PR TITLE
Updates GraphQL related dependencies

### PR DESCRIPTION
**Pull request by orta**
I ran locally, and everything seemed to work ok, obviously we don't really have a way of verifying _all_ requests, it has however been a ~year since I last updated these https://github.com/artsy/metaphysics/commit/d380856aece429b25bdcd747975f7aed5b175f3b

We're now triggering this warning on every test run / launch, https://github.com/graphql/graphql-js/blob/master/src/utilities/assertValidName.js#L40 so I've currently ignored it - I know you've done a bunch of work there anyway so I think ignoring it i tests is fine, but it shows when you launch the server

### CODE REVIEW

**Comment by orta** [comment]
TBH though, I just wanted the ability to command-click in GraphiQL to see docs

File: test/helper.js [hunk 1]
```diff
@@ -3,6 +3,11 @@
 import dotenv from "dotenv"
 dotenv.config({ path: ".env.test" })
 
+// Because we use `__id` as a UUID identifier, then
+// we have to tell GraphQL that we don't need a
+// warning about our usage of this.
+process.env.GRAPHQL_NO_NAME_WARNING = true
```
Comment (alloy) [inline test/helper.js]: Oh dope, there’s an env var for it?!

Comment (orta) [reply test/helper.js]: Yeah man, went and read the source 👍

**Comment by alloy** [comment]
This is a good idea, we’ve been behind by too much. However, let’s keep this one in mind for when/if memory starts spiking again.

**Pull request merged**